docs/api-documentation-en.md

```diff
@@ -0,0 +1,109 @@
+Heureka 'Ověřeno zákazníky' (ShopCertification) API Documentation
+============================================
+
+Documentation for [Heureka Ověřeno zákazníky](http://overeno.heureka.cz/) (ShopCertification) API
+service endpoint.
+
+Basics
+------
+
+To report an order created in your e-shop to the Heureka ShopCertification service, you have to make a HTTP POST
+request to the API endpoint URL with JSON-serialized data payload that describe the e-shop order (payload fields are
+described below).
+
+URL format
+----------
+
+Request URL has the following format:
+```https://api.heureka.cz/shop-certification/v2/[ACTION]```
+
+**Please note:** There are two available API endpoints. One is for Czech Heureka and the other one for Slovak Heureka
```

Comment:
prosimte zkrat ty texty, aby to pak nehazelo horizontalni scrollbar viz https://github.com/filipmelik/overeno-zakazniky/blob/c9cfe0bf46c8cbd1f1d3c3524d58c4cb3aebcd04/docs/api-documentation-en.md